classes/Language.php

```diff
@@ -1208,7 +1211,10 @@ public static function downloadAndInstallLanguagePack($iso, $version = _PS_VERSI
         return count($errors) ? $errors : true;
     }
 
-    public static function downloadLanguagePack($iso, $version, &$errors = [])
+    /**
+     * @param mixed $version Unused parameter. @deprecated since 8.0.0 and will be removed in next major version.
+     */
+    public static function downloadLanguagePack($iso, $version = null, &$errors = [])
```

Comment:
Deprecate the parameter version (if $version != null)

https://devdocs.prestashop.com/1.7/development/coding-standards/#deprecations